tests/integ/modin/frame/test_aggregate.py

```diff
@@ -162,6 +172,99 @@ def test_agg_dup_col(numeric_native_df, func, expected_union_count):
         eval_snowpark_pandas_result(snow_df, numeric_native_df, func)
 
 
+def test_named_agg_dup_col_pandas_fails_snowpark_pandas_supports(numeric_native_df):
```

Comment:
Can you split this test up or parametrize it so one failure doesn't cause the whole rest of the test to be skipped?